bots/images/scripts/fedora.setup

```diff
@@ -164,23 +175,26 @@ if ! [ "$version" -eq "$version" ] 2>/dev/null; then version=30; fi
 su builder -c "/usr/bin/mock --no-bootstrap-chroot --verbose -i $(/var/lib/testvm/build-deps.sh "$opsys $version")"
 su builder -c "/usr/bin/mock --install --verbose rpmlint"
 
-# HACK: docker falls over regularly, print its log if it does
-systemctl start docker || journalctl -u docker
-
-# our cockpit/base container is only really a thing on x86_64, just skip it on other arches
-if [ $(uname -m) = x86_64 ]; then
-    docker build -t cockpit/base /var/tmp/cockpit-base
+# HACK docker not available on f31
+# https://github.com/cockpit-project/cockpit/issues/12670
+if [ "$HAVE_DOCKER" ]; then
```

Comment:
not that important here, but let's please use a proper operator: `[ -n "$HAVE_DOCKER" ]`. Also, no need to repeat the comment.